test/helpers/test.go

```diff
@@ -42,6 +43,13 @@ import (
 	scmFactory "github.com/jenkins-x/go-scm/scm/factory"
 )
 
+const (
+	// BDDPRApproverUsernameEnvVar is the environment variable that we look at for the username for the approver in some tests
+	BDDPRApproverUsernameEnvVar = "BDD_APPROVER_USERNAME"
```

Comment:
So to run this test locally I will need to set these values as well now?

Reply:
If you want to have the `/approve` behavior fire - if they’re not set, it’ll still use `—auto-merge`. For the Lighthouse tests I’m working on now, yes, you will need a second user, but those tests will really only be relevant for Lighthouse PRs.